src/main/java/org/example/daos/JobRoleDao.java

```diff
@@ -98,4 +100,57 @@ public JobRole getJobRoleById(final int id) throws SQLException {
         }
     }
 
+    public int createJobRole(final JobRoleRequest jobRole) throws SQLException {
+        Connection connection = DatabaseConnector.getConnection();
+
+        String insertStatement =
+                "INSERT INTO job_roles (roleName, description, sharepointUrl, "
+                        + "responsibilities, numberOfOpenPositions, "
+                        + "location, closingDate, "
+                        + "capabilityId, bandId, statusId) "
+                        + "VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, 1)";
+
+        final int roleNameIndex = 1;
```

Comment:
move out and make static so that these are only ever created once - more efficient